pkg/api/v1/types.go

```diff
@@ -3404,6 +3404,15 @@ type PodExecOptions struct {
 	Command []string `json:"command" protobuf:"bytes,6,rep,name=command"`
 }
 
+// PodPortForwardOptions is the query options to a Pod's port forward call.
```

Comment:
Message should give more context about how port forward works - this will then show up in the public API docs.